packages/automerge-repo/test/helpers/collectMessages.ts

```diff
@@ -0,0 +1,19 @@
+import { EventEmitter } from "eventemitter3"
+import { pause } from "../../src/helpers/pause.js"
+
+export async function collectMessages({
+  emitter,
+  event,
+  until = pause(100),
```

Comment:
Rather than passing a timeout in ms, this gives the option of collecting messages until some other event occurs, e.g. 

```ts
await collectMessages({
  emitter: aliceTab2.networkSubsystem,
  event: "message",
  until: aliceTab2DocA.whenReady(),
})
```

Reply:
Nice!